diff --git a/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt b/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
index c2683dc..e50a7c2 100644
--- a/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
+++ b/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
@@ -1,9 +1,9 @@
 "UnlitGeneric" {
-	$basetexture "vgui/replay/thumbnails/speedo/digits"
-	// $basetexture "vgui/replay/thumbnails/speedo/digits_trebuchet"
+	// $basetexture "vgui/replay/thumbnails/speedo/digits"
+	$basetexture "vgui/replay/thumbnails/speedo/digits_trebuchet"
 
 	$translucent 1 $vertexcolor 1 $no_fullbright 1 $ignorez 1
 	
-	// $color "[1 0.8 0.8]"
+	$color "[1 0.8 0.8]"
 	// $alpha 0.75
 }
diff --git a/speedo/resource/speedo_absolute_center.res b/speedo/resource/speedo_absolute_center.res
index ff6d7df..43d673f 100644
--- a/speedo/resource/speedo_absolute_center.res
+++ b/speedo/resource/speedo_absolute_center.res
@@ -3,43 +3,43 @@
 		"controlName" "EditablePanel"
 		"visible" "1" // default visibility
 		"enabled" "1"
-		"wide" "40"
-		"tall" "10"
-		"xpos" "c-20"
-		"ypos" "c80"
-		
+		"wide" "43"
+		"tall" "11"
+		"xpos" "cs-0.5"
+		"ypos" "c100"
+
 		"d4_4k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d4_4k"
 		}
 		"d3_4k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d3_4k"
-			
+
 			"pin_to_sibling"		"d4_4k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
 		}
 		"d2_4k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d2_4k"
-			
+
 			"pin_to_sibling"		"d3_4k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
 		}
 		"d1_4k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d1_4k"
-			
+
 			"pin_to_sibling"		"d2_4k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
@@ -47,27 +47,27 @@
 
 		"d3_3k" {
 			"controlName" "CTFImagePanel"
-			"xpos" "5"
-			"wide" "10"
-			"tall" "10"
+			"xpos" "s0.5"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d3_3k"
 		}
 		"d2_3k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d2_3k"
-			
+
 			"pin_to_sibling"		"d3_3k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
 		}
 		"d1_3k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d1_3k"
-			
+
 			"pin_to_sibling"		"d2_3k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
@@ -75,17 +75,17 @@
 
 		"d2_2k" {
 			"controlName" "CTFImagePanel"
-			"xpos" "10"
-			"wide" "10"
-			"tall" "10"
+			"xpos" "s1.0"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d2_2k"
 		}
 		"d1_2k" {
 			"controlName" "CTFImagePanel"
-			"wide" "10"
-			"tall" "10"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d1_2k"
-			
+
 			"pin_to_sibling"		"d2_2k"
 			"pin_corner_to_sibling"		"7"
 			"pin_to_sibling_corner"		"5"
@@ -93,10 +93,10 @@
 
 		"d1_1k" {
 			"controlName" "CTFImagePanel"
-			"xpos" "15"
-			"wide" "10"
-			"tall" "10"
+			"xpos" "s1.5"
+			"wide" "11"
+			"tall" "11"
 			"image" "replay/thumbnails/speedo/absolute/center/d1_1k"
 		}
-	} 
-}
\ No newline at end of file
+	}
+}
