diff --git a/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt b/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
index c2683dc..e50a7c2 100644
--- a/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
+++ b/speedo/materials/vgui/replay/thumbnails/speedo/mod.vmt
@@ -1,9 +1,9 @@
 "UnlitGeneric" {
-	$basetexture "vgui/replay/thumbnails/speedo/digits"
-	// $basetexture "vgui/replay/thumbnails/speedo/digits_trebuchet"
+	// $basetexture "vgui/replay/thumbnails/speedo/digits"
+	$basetexture "vgui/replay/thumbnails/speedo/digits_trebuchet"
 
 	$translucent 1 $vertexcolor 1 $no_fullbright 1 $ignorez 1
 	
-	// $color "[1 0.8 0.8]"
+	$color "[1 0.8 0.8]"
 	// $alpha 0.75
 }
diff --git a/speedo/resource/speedo_absolute_center.res b/speedo/resource/speedo_absolute_center.res
index ff6d7df..62a2ff4 100644
--- a/speedo/resource/speedo_absolute_center.res
+++ b/speedo/resource/speedo_absolute_center.res
@@ -6,7 +6,7 @@
 		"wide" "40"
 		"tall" "10"
 		"xpos" "c-20"
-		"ypos" "c80"
+		"ypos" "c100"
 		
 		"d4_4k" {
 			"controlName" "CTFImagePanel"
